docs/example.rst

```diff
@@ -0,0 +1,70 @@
+###############
+Running Example
+###############
+
+You can find examples of BEAST runs in the ``beast/examples/`` directory.
+
+Inside each example, there is a run_beast*.py script.
+
+phat_small Example
+------------------
+
+This example is based on a *very* small amount of old PHAT data.
+
+If the beast has not been installed (only downloaded from github), then
```

Comment:
I think this should be removed.  I don't like the soft link way of doing this.  We should just say you should install the beast and point the user to the install instructions.